cocos/3d/CCSprite3D.h

```diff
@@ -200,6 +200,21 @@ class CC_DLL Sprite3D : public Node, public BlendProtocol
     */
     void setForce2DQueue(bool force2D);
 
+    /**
+    * Get meshes used in sprite 3d
+    */
+    const Vector<Mesh*>& getMeshes() const { return _meshes; }
+
+    /*
+    * Get AABB
+    * If the sprite has animation, it can't be calculated accurately,
+    * because bone can drive the vertices, we just use the origin vertices
+    * to calculate the AABB.
+    *
+    * @param world Use world transform or not.
+    */
+    const AABB& getAABB(bool world) const;
```

Comment:
The old one is const AABB& getAABB(bool world) const. I do not think this modification is necessary.